CHANGELOG.md

```diff
@@ -1,9 +1,9 @@
 # Upcoming
 
 * BREAKING CHANGE: Seperate the code from the angular_token_auth into seperate files.
-* Add file linting.
```

Comment:
I thought changelog is just for source changes? I.e. anyone who's reading it wants to find out how the package has changed in their installation?